core/src/Cuda/Kokkos_Cuda_ReduceScan.hpp

```diff
@@ -62,33 +62,37 @@ template <class ValueType, class ReducerType>
 __device__ inline void cuda_inter_warp_reduction(
     ValueType& value, const ReducerType& reducer,
     const int max_active_thread = blockDim.y) {
-#define STEP_WIDTH 4
+#define KOKKOS_IMPL_STEP_WIDTH 4
```

Comment:
I don't see a good reason to use a macro here in the first place.
```suggestion
constexpr int step_width = 4;
```

Reply:
Changed.